app/views/contestresult.scala.html

```diff
@@ -157,19 +157,19 @@ <h4>コンテスト結果</h4>
         <table class="table table-bordered result tribox-contest">
             <thead>
                 <tr>
-                    <th>順位</th>
-                    <th>獲得SP</th>
+                    <th><i class="fa fa-hashtag"></i></th>
+                    <th>SP</th>
                     <th>名前</th>
                     <th class="mb-hide">所属</th>
                     @if(eid != "333fm") {
-                        <th ng-if="events.e@{eid}.method == 'average' || events.e@{eid}.method == 'mean'">平均記録</th>
-                        <th ng-if="events.e@{eid}.method == 'best'">最高記録</th>
-                        <th class="mb-hide">個々の記録</th>
+                        <th ng-if="events.e@{eid}.method == 'average' || events.e@{eid}.method == 'mean'">記録</th>
+                        <th ng-if="events.e@{eid}.method == 'best'">記録</th>
+                        <th class="mb-hide"></th>
                     } else {
                         <th>記録</th>
-                        <th class="mb-hide">解答・解説コメント</th>
+                        <th class="mb-hide"></th>
```

Comment:
164-172行目も条件分岐の必要性がなくなったので、
```suggestion
                        <th>記録</th>
                        <th class="mb-hide"></th>
```
とまとめちゃってよさそうです。